app/models/spree/user_decorator.rb

```diff
@@ -1,20 +1,23 @@
 module Spree::UserDecorator
 
   def self.prepended(base)
-    base.after_create :create_mollie_customer
+    base.after_save :ensure_mollie_customer
```

Comment:
The after save method is only triggered when the user itself is updated. If the user record is touched by for example an associated order it will not be called. An after_commit might be better suited.

Reply:
In addition, when a user is deleted we might want the mollie customer deleted too in the GDPR scale of things?

Reply:
Agreed, but that might be something for a separate PR (just to not introduce something new)